src/main/resources/edu/cornell/kfs/module/cg/businessobject/datadictionary/AwardAccountExtendedAttribute.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:p="http://www.springframework.org/schema/p"
+	xmlns:dd="http://rice.kuali.org/dd"
+	xsi:schemaLocation=" http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd http://rice.kuali.org/dd http://rice.kuali.org/dd/dd.xsd">
+
+	<bean id="AwardAccountExtendedAttribute" parent="AwardAccountExtendedAttribute-parentBean" />
+	<bean id="AwardAccountExtendedAttribute-parentBean" abstract="true" parent="BusinessObjectEntry">
+		<property name="businessObjectClass" value="edu.cornell.kfs.module.cg.businessobject.AwardAccountExtendedAttribute" />
+		<property name="objectLabel" value="AwardAccountExtendedAttribute" />
+		<property name="attributes">
+			<list>
+				<ref bean="AwardAccountExtendedAttribute--proposalNumber" />
```

Comment:
there is an extra dash between AwardAccountExtendedAttribute and proposalNumber. Has this maybe cause some issues?

Reply:
ok, will fix